src/main/java/com/techiteasy/demo/models/Television.java

```diff
@@ -38,35 +38,36 @@ public class Television {
     //Data worden opgeslagen als DATE of TIMESTAMP types in de database
     //Enums worden opgeslagen als VARCHAR of ENUM type in de database
 
-    //Default constructor - dit is handig wanneer je een object wilt aanmaken zonder onmiddellijk waarden toe te wijzen aan de velden.
-    public Television() {
+    //OneToOne-relatie met RemoteController
```

Comment:
ik zie hier dat de owner is verschillend in elke entiteit relatie, het is niet erg maar het is wel netter om te zien dat je een aanpakt heeft.

```suggestion
    Hoofd tabel zet ik altijd als een non-owning tabel. Deze manier krijgt die hoofdtabel niet de extra velden.
```